src/api/register.js

```diff
@@ -14,7 +14,7 @@ module.exports = async (dvf, starkPublicKey) => {
 
   const { deFiSignature } = await post(url, { json: data })
 
-  const onchainRegister = true //await dvf.stark.register(dvf, starkKey, deFiSignature)
+  const onchainRegister = await dvf.stark.register(dvf, starkKey, deFiSignature)
```

Comment:
So is this

Reply:
Same for this one. Rebased so this is not coming up as a change now.